### PR TITLE
feat: hype the docs (SRC-638)

### PR DESCRIPTION
**Pull request by dominikdotsat**
This PR adds new appearance for the docs, that more in line with our current branding, probably we will adjust it more with design. 

(had a few template limitation, so some of the element were hided by styling, some changed in the configuration)

moslty css change, I was not able to call the @motain/web-area, just Andrea & Mario 😅

### CODE REVIEW

**Comment by davcevski** [comment]
I think its worth minifying the assets here. Have you considered that?

**Comment by ponkys** [comment]
I think it is a step that is not needed in this type of resources and will just add more steps into the project building. I will prefer simplicity over performance in this case. Just my opinion

**Comment by ponkys** [comment]
I will delete this file as well https://github.com/motain/creators-network-docs/blob/master/source/codeexample.py

**Comment by dominikdotsat** [comment]
> I would delete this file as well https://github.com/motain/creators-network-docs/blob/master/source/codeexample.py

the naming is misleading, it's not a codexample for this type of docs, the docs really have codexample inside of it :D

File: source/_static/main.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@import url('style/colors.css');
+@import url('style/fonts.css');
+@import url('style/typography.css');
+@import url('style/codeexample.css');
+
+code.docutils.literal.notranslate {
+  background-color: var(--brand-green);
+}
+
+div.document {
```
Comment (ponkys) [inline source/_static/main.css:12]: I couldn't figure out why is there a div before the document or body? What is this jsx?

Comment (dominikdotsat) [reply source/_static/main.css:12]: it's needed to overwrite current css rules

File: source/_static/style/fonts.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  font-display: swap;
+  font-weight: 500;
+  font-style: normal;
+  src: url('https://s3-eu-west-1.amazonaws.com/static.onefootball.com/.dodgercms/fonts/formular-medium.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Formular bold';
+  font-display: swap;
+  font-weight: bold;
```
Comment (Bosc85) [inline source/_static/style/fonts.css:20]: just double checking, are you sure is needed to specify font-weights? the font files should contain only one variant

Comment (dominikdotsat) [reply source/_static/style/fonts.css:20]: yeah, you're right, we don't need them 😅

**Pull request merged**